src/wallet/index.ts

```diff
@@ -152,8 +158,12 @@ class Wallet {
     transaction: Transaction,
     options: SignOptions = { signAs: '' },
```

Comment:
We should be able to get rid of `SignOptions` now

Reply:
Since we only need an optional `signAs`, you could make just that an optional param, or rename it to something like `multisignSigner` (so that it's more clear that it's just for multisign)